1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,52 @@
 function readingList(books) {
-  // Write your code here...
+  // Write your code here..
+  const getParentElement = document.querySelector("#content");
```

Comment:
You can use document.getElementById here, and maybe shorten the varible name to parentElement getParentElement implies that it's a function to get the parent element when it is in fact the parent element